### PR TITLE
updated and cleaned up some documentation

### PR DESCRIPTION
**Pull request by sgpthomas**
Cleaned up some documentation

### CODE REVIEW

**Comment by sgpthomas** [comment]
We should probably add a section on dual ported memories

File: docs/cheat.md [hunk 1]
```diff
@@ -189,76 +189,59 @@ import "printer.h" {
...
-See the views doc for the semantics of `split`.
-
-Split views are created using the following syntax:
-
-```C
-decl a: bit<32>[16 bank 8];
-split v = a[by <factor>]
+```dahlia
+let A: float[8];
+view split_A = split A[by 2]; // split_A has type float[4 bank 2][2 bank 2];
```
Comment (sampsyo) [inline docs/cheat.md]:
```suggestion
view split_A = split A[by 2]; // split_A has type float[4 bank 2][2 bank 2]
```

File: docs/cheat.md [hunk 2]
```diff
@@ -189,76 +189,59 @@ import "printer.h" {
...
+```dahlia
+let A: float[6 bank 4];
+for (let i = 0..4) {
+  view s = shift A[by i * i];
+}
 ```
 
-### Split views
+### Split Views
+Split views transform  single dimension of a memory into a multi-dimensional memory.
```
Comment (sampsyo) [inline docs/cheat.md]:
```suggestion
Split views transform one dimension of a memory into multiple dimensions.
```

File: docs/overview.md [hunk 3]
```diff
@@ -44,17 +44,17 @@ researchers to explore this problem.
...
+[**Dahlia**][repo]: The language constructs and the reference compiler. We are
 focusing our efforts on building precise semantics and compiling
 down to various HLS backends to study their properties.
 
 [**FuTIL**][futil-repo] (Fuse Temporal Intermediary Language): An intermediary language for compiling high level
 languages down to RTL. Distinct from IRs like [FIRRTL][firrtl] which target the lower level problem of optimizing RTL code.
 
-**Fuse-to-RTL**: Ongoing project to directly generate RTL code from Fuse source
+**Dahlia-to-RTL**: Ongoing project to directly generate RTL code from Dahlia source
 programs using FuTIL.
```
Comment (sampsyo) [inline docs/overview.md]:
```suggestion
programs using [FuTIL](https://github.com/cucapra/futil).
```

File: docs/cheat.md [hunk 4]
```diff
@@ -189,76 +189,59 @@ import "printer.h" {
...
+for (let i = 0..4) {
+  view s = suffix A[by 2 * i];
+  ---
+  view s = prefix A[by 2 * i];
+}
 ```
 
-Example:
+### Shift Views
+These are generalized suffix / prefix views that lift the restriction that the banking factor
```
Comment (sampsyo) [inline docs/cheat.md]:
```suggestion
These are generalized suffix/prefix views that lift the restriction that the banking factor
```

**Comment by rachitnigam** [comment]
merge whenever the suggested changes are fixed. The website will autobuild.

**Pull request merged**